pkg/context/context.go

```diff
@@ -0,0 +1,64 @@
+package context
+
+import (
+	"chaos-mesh/matrix/pkg/node"
+	"chaos-mesh/matrix/pkg/node/data"
+	"chaos-mesh/matrix/pkg/random"
+	"chaos-mesh/matrix/pkg/serializer"
+	"fmt"
+)
+
+type MatrixContext struct {
+	Configs map[string]node.AbstractConfig
+}
+
+// This is to generate real value from an abstract tree
+func (c MatrixContext) Gen() node.ConfigGroup {
+	var result node.ConfigGroup
+
+	result.Configs = make(map[serializer.Config]interface{})
+
+	for _, config := range c.Configs {
+		result.Configs[config.Config] = simpleRecGen(config.Hollow)
+	}
+
+	return result
+}
+
+func simpleRecGen(hollow interface{}) interface{} {
```

Comment:
what means hellow

Reply:
I think it's "trick or treat" which generates random variables.